.goreleaser.yaml

```diff
@@ -89,3 +89,21 @@ docker_manifests:
   image_templates:
   - ghcr.io/tensorchord/envd-ssh-from-scratch:{{ .Version }}-amd64
   - ghcr.io/tensorchord/envd-ssh-from-scratch:{{ .Version }}-arm64v8
+brews:
+  - name: envd
+    ids:
+    - envd
+    tap:
+      owner: tensorchord
+      name: homebrew-tap
+    commit_author:
+      name: Ce Gao
+      email: cegao@tensorchord.ai
```

Comment:
Is it possible to use org as the commit author here?

Reply:
Sure. This section just sets the username and email of git in commit messages.

Reply:
Then let's use TensorChord and envd-maintainers@tensorchord.ai here

Reply:
Got it.